### PR TITLE
313 objectives

### PR DESCRIPTION
**Pull request by mona12rst**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -71,4 +71,5 @@ crashlytics-build.properties
 # Temporary auto-generated Android Assets
 /[Aa]ssets/[Ss]treamingAssets/aa.meta
 /[Aa]ssets/[Ss]treamingAssets/aa/*
+omnisharp.json
```
Comment (Strike2610) [inline .gitignore:74]: we added this for consistancy, pls no remove

File: Assets/Scripts/Logic/Player.cs [hunk 2]
```diff
@@ -39,7 +39,7 @@ public void OnEnergyAdded(float currentAmount)
 		} else {
 			Instance = this;
 		}
-		Energy = ES3.Load("Energy", 2000f);
+		Energy = ES3.Load("Tymo", 2000f);
```
Comment (Strike2610) [inline Assets/Scripts/Logic/Player.cs:42]: Not for actual pushes

File: .gitignore (hunk 1 above)
Comment (abbydiode) [inline .gitignore:74]: Can we please keep this for consistency sake?

File: omnisharp.json [hunk 3]
```diff
@@ -2,18 +2,22 @@
...
-		"NewLineForElse": false,
-		"NewLineForCatch": false,
-		"NewLineForFinally": false
+		"NewLineForElse": true,
+		"NewLineForCatch": true,
+		"NewLineForFinally": true,
+		"NewLineForClausesInQuery": true,
+		"IndentSwitchCaseSection": true
+		
+
```
Comment (abbydiode) [inline omnisharp.json:21]: You are a psychopath

File: omnisharp.json [hunk 4]
```diff
@@ -2,18 +2,22 @@
...
 
-		"NewLineForElse": false,
-		"NewLineForCatch": false,
-		"NewLineForFinally": false
+		"NewLineForElse": true,
+		"NewLineForCatch": true,
+		"NewLineForFinally": true,
+		"NewLineForClausesInQuery": true,
+		"IndentSwitchCaseSection": true
+
```
Comment (Strike2610) [inline omnisharp.json:20]: this is a redundantly verbose form of writing code

**Review by Strike2610 — CHANGES_REQUESTED** [review]
abby told me to!

**Review by abbydiode — APPROVED** [review]

**Pull request merged**